src/lib/components/workflow/start-workflow-search-attribute-input.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import Button from '$lib/holocene/button.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import NumberInput from '$lib/holocene/input/number-input.svelte';
+  import Option from '$lib/holocene/select/option.svelte';
+  import Select from '$lib/holocene/select/select.svelte';
+  import type {
+    SearchAttributeInput,
+    SearchAttributeOption,
+  } from '$lib/stores/search-attributes';
+
+  export let attributesToAdd: SearchAttributeInput[] = [];
+  export let searchAttributes: SearchAttributeOption[] = [];
+  export let attribute: SearchAttributeInput;
+  export let onRemove: (attribute: string) => void;
+
+  $: type =
+    searchAttributes.find((a) => a.value === attribute.attribute)?.type ?? '';
+
+  const clearValue = (_) => (attribute.value = '');
+
+  $: clearValue(type);
```

Comment:
I had to squint to figure out what you're doing here, but it seems a bit too clever.